code/modules/surgery/organs/organ_internal.dm

```diff
@@ -15,6 +15,12 @@
 		insert(holder)
 	..()
 
+
+/obj/item/organ/internal/process()
```

Comment:
https://github.com/Aurorablade/Paradise/blob/BeepBeep/code/modules/surgery/organs/organ_internal.dm#L49-L66

These lines should |= into `processing_objects`.

https://github.com/Aurorablade/Paradise/blob/BeepBeep/code/modules/surgery/organs/organ_internal.dm#L24-L46
These lines should -= from `processing_objects`